### PR TITLE
Added branding and fixed copyright

### PR DESCRIPTION
**Pull request by asutoshvariar**
Added current Ducktape logo, created Ducktape favicon and fixed the Copyright year issue.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ const config = {
             ],
           },
         ],
-	copyright: `Copyright &copy; {new Date().getFullYear()} Ducktape. Built with Docusaurus.`
+	copyright: 'Copyright &copy; ' + new Date().getFullYear() + ' Ducktape. Built with Docusaurus.'
```
Comment (lines-of-codes) [inline docusaurus.config.js:101]: I was wondering, Why did you need to change this?

**Review by lines-of-codes — COMMENTED** [review]
Everything seems good! But my curious mind is wondering about why did you need to change from template string to a normal string with concatenation.

File: docusaurus.config.js (hunk 1 above)
Comment (lines-of-codes) [reply docusaurus.config.js:101]: OK actually my mind just forgot how JavaScript work for a second

**Comment by asutoshvariar** [comment]
I don't know, honestly! The {new Date().getFullYear()} section was just showing up as plain text for me and the normal string method fixed it. I am using Firefox (macOS), so maybe that makes a difference?
